### ISSUE TITLE
Cannot access the nested JSON Object

### ISSUE THREAD

**Issue by powercoder23** [ocombe/ng2-translate#15]
I am facing an issue, when i am using single string value in JSON it works, but when using object instead it doesn't works, below is the code explained.

```typescript
   translate.setDefaultLang('en');

    translate.setTranslation('en', {
        "helloWorld": {
             "sayHello": "Hello there!"
         },
         "helloWorldSayHello": "Hello there!"
    });

    this.translate.use('en');
```
```HTML
    <div>{{ 'helloWorldSayHello' | translate }} world</div> ----------//this does work
    <div>{{ 'helloWorld.sayHello' | translate }} world</div> ----------//this doesn't work
```

Is their any other way to make it work?

**Comment by ocombe** [comment]
Not yet, but it's just a matter of improving the parser, I'll put it on my todo list :)

**Comment by webprofusion-chrisc** [comment]
:+1: I'm currently looking to rebuild an existing app in Ionic2/Angular2 - our translations are created using https://webtranslateit.com/en and the resulting json looks like this: https://github.com/openchargemap/ocm-system/blob/master/Localisation/src/OCM_UI_LocalisationResources.it.json

When the translation for the selected language is null, we fallback (currently to the default text in the app). It would also be great to specify a fallback language in config. So if your app is in English but is mostly translated to Japanese then you could fallback to English.

**Comment by powercoder23** [comment]
Thank you @ocombe ! you are doing a great job :+1:

**Comment by ocombe** [comment]
Both features are available in 1.4.0 !

**Comment by WillGeller** [comment]
Still having issues with attempting to use a nested json source for translations.
`<div>{{ 'helloWorld.sayHello' | translate }}</div>
`
displays `[Object object]` rather then the translation value
